JASP-Engine/JASP/R/descriptives.R

```diff
@@ -83,7 +83,7 @@ Descriptives <- function(jaspResults, dataset, options) {
   if (options$plotVariables) {
     if(is.null(jaspResults[["distributionPlots"]])) {
       jaspResults[["distributionPlots"]] <- createJaspContainer(gettext("Distribution Plots"))
-      jaspResults[["distributionPlots"]]$dependOn(c("plotVariables", "splitby", "distPlotDensity"))
+      jaspResults[["distributionPlots"]]$dependOn(c("plotVariables", "splitby", "distPlotDensity", "distPlotRug"))
```

Comment:
As with the breaks, the histograms of categorical variables don't change when you change add rugs. So that dependency should be specified elsewhere.